ci/apache-arrow.rb

```diff
@@ -0,0 +1,75 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
```

Comment:
I think that we should not relicense this file. Because we will send back the changes of this file to Homebrew. If we relicese this file to Apache License 2.0, we need to relicense this to BSD 2-Clause License again when we creating a pull request. I'm not sure but it'll require vote or something.

@wesm Do you know any workflow when we send a patch to non ASF projects from our repository?

Reply:
Can I just ignore this file in the RAT check?

Reply:
For the "vendored" conda-recipes we have a RAT [exclusion](https://github.com/apache/arrow/blob/master/dev/release/rat_exclude_files.txt#L133), and a license entry [referring the original license](https://github.com/apache/arrow/blob/master/LICENSE.txt#L744). I suggest to do the same.